roles/software_engineer_2.md

```diff
@@ -2,17 +2,61 @@
 
 Bristol, Manchester & London, UK.
 
-Our Engineers build software that makes our customers happy, they prefer problem solving over task completing, love learning from colleagues, and constantly strive to be nice humans.
+Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
+
+Our Engineers deliver digital, data and technology outcomes that improve society. They do this by delivering and architecting software.
 
 ## What does the job entail?
 
-We primarily write and deliver custom software for the public sector. We work across central and local government, as well as in health, and our past lies in the technology startup world. Technical excellence for us isn’t about delivering to feature lists. We place a strong emphasis on outcome-based delivery; ensuring our customer’s goals are understood and achieved with the technology we deploy.
+At Made Tech we want to positively impact the future of the country by using technology to improve society. We help public sector organisations deliver quality software to help citizens get more from public services. To do this we work alongside brilliant public servants to modernise technology and accelerate digital delivery.
+
+As an Engineer 2 you will collaboratively lead planning of work, seeking feedback and support when needed, influence the approach taken to deliver features for our customers and their users, and will be comfortable managing complex changes within a single system.
+
+You will have the opportunity to work across a variety of technologies and will need to be comfortable doing so. Our teams have used Ruby with Rails and Sinatra, ES6 with React and Angular, C# with .NET Core, Java with Spring, Python with Django, and plenty more stacks. We don’t limit ourselves as a company to particular technologies and we expect all our engineers to be keen on learning new skills.
+
+High performing software delivery teams need to be empowered to iteratively and rapidly deliver changes all the way through to production. To do this we combine our extensive cloud automation knowledge with DevOps culture.
+
+We've been using AWS from the start and as Advanced Partners are go to experts within the public sector. We use a range of IaaS, PaaS and FaaS depending on the needs of our users, in this case software teams, such as EC2, Lambda, ECS, Kubernetes, Heroku, CloudFoundry, Azure App Services, and more. We use VPC and PrivateLink for connecting to on-premise, legacy systems. We also use API Gateway, S3, CloudFront, SQS, SNS, SES, RDS, and many other services provided by AWS. You will have the opportunity to use and learn these technologies and we offer official AWS and other cloud training.
```

Comment:
This and the following two paragraphs are for a DevOps centric role - I don't think this is the case generally for Engineer 2, though might be relevant for roles we're currently hiring for?

Reply:
Great, be good to see what Hiring are looking for and to clarify - would Engineers from the academy be given some DevOps experience I guess?

Reply:
Almost certainly, on reflection I think it's actually the last paragraph that set off this comment, will comment there separately